spec/pagseguro/subscription_spec.rb

```diff
@@ -69,5 +114,24 @@
         expect { subject.create }.to change { subject.code }.to('12345')
       end
     end
+
+    context 'find subscriptions' do
+      let(:code) { 12345 }
+      let(:request) { double(:Request, success?: true, xml?: true, body: '') }
+
+      context 'by notification code' do
+        let(:search) { PagSeguro::Subscription.find_by_notification_code(code, credentials: credentials) }
+        path = 'v2/pre-approvals/notifications/12345?email=user@example.com&token=TOKEN'
```

Comment:
It looks strange creating a variable right here…

I would have liked something like this:

```
it_behaves_like 'Subscription find', 'v2/pre-approvals/notifications/12345?email=user@example.com&token=TOKEN'
```